src/interactions/map_arcade/arcade_employee.interaction.js

```diff
@@ -0,0 +1,12 @@
+import { firstConversationArcade } from "../../constants";
+import { displayDialogue } from "../../utils";
```

Comment:
first, please pull the latest changes from the main branch.
Then, replace the `displayDialoge`function import with the new function `displayDialogueWithCharacter` which you can find [here](https://github.com/zero-to-mastery/ZTM-Quest/blob/c6296090a3dc8b0130fdb72af1458d66d8e7a4cb/src/utils.js#L35) as a reference.
This new function accepts the character name as first parameter. The rest is the same.
You can get creative and give the aracde_employee a fancy name

Reply:
Changes done.
src/interactions/map_start/jokeTeller.interaction.js I am not created this file it give some lint issues.